docs/cardinal/game/query.mdx

```diff
@@ -68,3 +76,26 @@ You can easily create a new query and register it to the world by following thes
 </Steps>
 
 --- 
+
+## Query Options
+
+### EVM Support
+
+Queries can be sent by EVM smart contracts by using the `WithQueryEVMSupport` option when you register your queries. This will generate the ABI types necessary for interactions with smart contracts.
+
+
+```go
+import (
+    "pkg.world.dev/world-engine/cardinal"
+    qry "pkg.world.dev/world-engine/cardinal/query"
```

Comment:
@technicallyty should we find a different name for this package so we can avoid this

Reply:
Perhaps we can rename the `query/` dir in starter game template to `qry/`, since we're already using `msg/` for messages.

Reply:
So wdyt @smsunarto? should we go with this? if so we'll need to change other parts of the doc as well as sgt

Reply:
This is a todo, just ignore it for this or [](edited-with-Axolo)